inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/responses/ping/CampaignData.kt

```diff
@@ -32,4 +32,10 @@ internal data class CampaignData(
     override fun isTest(): Boolean = isTest
 
     override fun getMaxImpressions(): Int = maxImpressions
+
+    override fun getContexts(): List<String> {
```

Comment:
Why use `List`, will we handle/consider multiple contexts per campaign?

Reply:
Yes. I thought that it might be a requirement in the future. For example user might want to include various id of screens that will be applicable for that campaign